internal/test/scorecard/common_utils.go

```diff
@@ -511,11 +513,33 @@ func (r *TestResources) sendHealthRequest(base *url.URL, healthCheck func(resp *
 	return err
 }
 
-func (r *TestResources) updateAndWaitTillCryostatAvailable(cr *operatorv1beta1.Cryostat) error {
-	cr, err := r.Client.OperatorCRDs().Cryostats(cr.Namespace).Update(context.Background(), cr)
-	if err != nil {
-		r.Log += fmt.Sprintf("failed to update Cryostat CR: %s", err.Error())
+func (r *TestResources) updateAndWaitTillCryostatAvailable(cr *operatorv1beta1.Cryostat) (*operatorv1beta1.Cryostat, error) {
+	ctx := context.Background()
+
+	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+		cr, err := r.Client.OperatorCRDs().Cryostats(cr.Namespace).Get(ctx, cr.Name)
```

Comment:
This line declares another `cr` variable within the scope of retry func. Thus, the returned `cr` still holds pointer to the empty CR struct.

Reply:
A simple fix can be:

```suggestion
        var err error
		cr, err = r.Client.OperatorCRDs().Cryostats(cr.Namespace).Get(ctx, cr.Name)
```